imports/plugins/core/graphql/server/schemas/role.js

```diff
@@ -1,14 +1,16 @@
 export const typeDefs = `
-  # RoleName - String that conforms to Role regex.
-  # This may be required if the role name must match a specific pattern.
-  # Could also be made more generic depending on pattern.
-  # Remove if not used.
-  scalar RoleName
+  # The fields by which you are allowed to sort any query that returns an \`RoleConnection\`
+  enum RoleSortByField {
+    _id
+    name
+    createdAt
+    updatedAt
```

Comment:
Does that collection have `createdAt` and `updatedAt`? If not, remove them from here and make `name` the default sort.

Reply:
No it does not, good catch.